config/api.yml

```diff
@@ -1476,6 +1488,10 @@
         :identifier: ems_infra_protect
       - :name: unassign
         :identifier: ems_infra_protect
+    :container_images_subcollection_actions:
```

Comment:
you shouldn't have to define these again here, unless they were specific to the collection. so defining them above would be just fine